src/main/java/com/deegeu/trivia/model/TriviaQuestionArrayAccess.java

```diff
@@ -86,7 +88,17 @@ public TriviaQuestion getRandomQuestion() {
 
     @Override
     public List<TriviaQuestion> getSpecifiedQuestionList(long... id) {
-        throw new UnsupportedOperationException("Not supported yet."); //To change body of generated methods, choose Tools | Templates.
+
+        // Convert the var args into a List
+        final List<Long> ids = Arrays.stream(id)
```

Comment:
varargs are arrays by default, but I wanted to convert them into a Collection so that I can use the `contains` method in the `filter` below.